include_core/unix/thrdsup.h

```diff
@@ -71,6 +71,15 @@ typedef WRAPPER_TYPE (*WRAPPER_FUNC)(WRAPPER_ARG);
 #include <semaphore.h>
 typedef sem_t OSSEMAPHORE;
 #elif defined(OSX) /* defined(LINUX) || defined(AIXPPC) */
+#if defined(__GNUC__)
+#ifndef __has_extension
```

Comment:
@babsingh Can you add a comment to this code that describes why this is required?

Reply:
added a comment